### ISSUE TITLE
Screen shuts off randomly while playing audio

### ISSUE THREAD

**Issue by deutrino** [signalapp/Signal-Android#6993]
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

Somewhat often lately when receiving an audio attachment, when I attempt to play it, my phone's display shuts off almost immediately. It's shutting all the way off, not just alive and colored black. Often it will come back to life. I haven't characterized well how often it's coming back to life in the app vs at the lock screen, I think I've seen both. This is not specific to the audio file as the behavior can come and go with a given attachment. I've not figured out what clears the problem. It will not clear on its own if you stay in the conversation view, but if you go and do some other things, the problem may clear and the file will play back correctly.

### Device info
**Device:** Moto X Pure Edition (clark)
**Android version:** 7.1.2 (lineageOS 14.1-20170904-NIGHTLY-clark)
**Signal version:** 4.10.12

### Link to debug log
https://gist.github.com/deutrino/9133b98e82c7fb27fe92935267dc9582

**Comment by FeuRenard** [comment]
My guess: your phone's proximity sensor is over-sensitive.

A feature of Signal voice note playback is that it reacts to the user holding the phone to his/her ear. Signal then uses the earpiece for playback and turns off the screen so your face won't operate the screen. As soon as you take the phone away from your ear the screen should be turned on again and if the voice note hasn't finished playing yet it will stop playing then.

Can you hear the voice note playback from the earpiece after your screen shut off while playing audio?

**Comment by deutrino** [comment]
I'm not 100.0% positive but I am pretty certain that I did hear playback from the earpiece at least in some cases.

However, this behavior started within the past couple months, and my phone exhibits zero problems like this in any other app. It's also happened with the phone sitting on a flat surface after I tapped the play button on the audio attachment and removed my finger from proximity.

I can do some additional tests to characterize the behavior if that would be helpful.
